src/Attestation/Attestation/Commands/GetAzureAttestation.cs

```diff
@@ -77,16 +77,6 @@ public override void ExecuteCmdlet()
                 ResourceGroupName = resourceIdentifier.ResourceGroupName;
             }
 
-            if (string.IsNullOrEmpty(Name))
-            {
-                throw new CloudException(string.Format("ResourceNotSpecified", Name));
-            }
-
-            if (string.IsNullOrEmpty(ResourceGroupName))
-            {
-                throw new CloudException(string.Format("ResourceGroupNotSpecified", ResourceGroupName));
-            }
-
```

Comment:
Could you please offer more info why delete these check? Do you make these parameter `required`?

Reply:
I don't delete the null check, just replace it with
            if (string.IsNullOrWhiteSpace(attestationName))
            {
                throw new ArgumentNullException("attestationName");
            }
            if (string.IsNullOrWhiteSpace(resourceGroupName))
            {
                throw new ArgumentNullException("resourceGroupName");
            }